modules/platforms/dotnet/Apache.Ignite.Tests/JavaServer.cs

```diff
@@ -33,17 +33,25 @@ public sealed class JavaServer : IDisposable
     {
         private const int DefaultClientPort = 10942;
 
-        private const int ConnectTimeoutSeconds = 20;
+        private const int ConnectTimeoutSeconds = 120;
```

Comment:
Is it necessary to increase the timeout so drastically? Does Gradle introduce some kind of a delay?

Reply:
Unlit we implement shared build cache for Gradle in CI some steps can produce additional build steps and this can increase start time. So, this is temporary change, until we do some devops magic.

Reply:
Actually, `EnsureBuild` method was added for this reason. Any building, cache warmup, etc should go there. It is not covered by the timeout and can take as long as needed.

However, node startup itself should have a reasonable timeout, because we want to detect issues with cluster connections.

So it would be great to reduce the timeout to the original value and do all the required heavy lifting in `EnsureBuild`.